VMDP doc set (#111284)
openshift/openshift-docs: modules/oadp-vmdp-overview.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmdp/oadp-vmdp.adoc

:_mod-docs-content-type: CONCEPT
[id="oadp-vmdp-overview_{context}"]

> = About {oadp-short} VM data protection

Reviewer: This appears to be only of the few headings that spells VMDP. For consistency I'd unify how we title files - either all use acronyms or full term.

Author: I am using the full name now everywhere at most places except for a few where the word count of the title is exceeding the permitted limit.

[role="_abstract"]

> {oadp-short} VM data protection (VMDP) enables virtual machine (VM) users to back up and restore their own data. Learn when to use VMDP and how it differs from traditional admin-managed backups.

Reviewer: As a new user, I wouldn't be able to figure out if VMDP is something I could benefit from. We could improve the short description by focusing on the user instead, WDYT?

```suggestion
You can independently back up and restore your own data from within a virtual machine (VM) by using the {oadp-short} VM data protection (VMDP) command-line tool. This approach helps you secure specific files and directories in your encrypted repository without requiring cluster administrator privileges.
```

296 characters

== What problem is VMDP solving

> Traditional {oadp-short} backups are managed by cluster administrators. The administrator owns the backup storage location, controls what is backed up, and manages the restore process. This means that VM users must rely on an administrator to recover their data, and the backup scope is limited to the PVCs attached to the VM at the time of backup.

Reviewer: ```suggestion
Cluster administrators manage traditional {oadp-short} backups. The administrator owns the backup storage location, controls what to back up, and manages the restore process. This means that VM users must rely on an administrator to recover their data, and the backup scope is limited to the persistent volume claims (PVCs) attached to the VM at the time of backup.
```

To avoid passive voice.

> VMDP addresses this gap by shifting data ownership to the VM user. The user creates their own encrypted backup repository, chooses what data to protect, and restores data independently without administrator involvement. This follows zero-trust architecture principles where the user owns the data, the backup, and the encryption keys. Administrators cannot access or restore the user's backup data.

Reviewer: ```suggestion
VMDP addresses this gap by shifting data ownership to the VM user. The user creates their own encrypted backup repository, chooses what data to protect, and restores data without administrator involvement. This follows zero-trust architecture principles where the user owns the data, the backup, and the encryption keys. Administrators cannot access or restore the user's backup data.
```

== What VMDP does

> VMDP is a command-line tool that runs inside virtual machines on {VirtProductName}. With VMDP, you can:

Reviewer: ```suggestion
VMDP is a command-line tool that runs inside virtual machines on {VirtProductName}. With VMDP, you can complete the following tasks:
```

It is not wrong, just full sentences work better.

* Back up and restore files and directories from within the VM by using a single command.
* Protect data accessible over network file systems such as Common Internet File System (CIFS) and Network File System (NFS) shares, which standard {oadp-short} backups typically exclude.

> * Create a personal encrypted repository in S3-compatible or filesystem storage.

Reviewer: ```suggestion
* Create a personal encrypted repository in S3-compatible or file system storage.
```

> * Use data de-duplication for efficient storage and fast incremental backups.

Reviewer: ```suggestion
* Use data deduplication for efficient storage and fast incremental backups.
```
